app/templates/gulp-tasks/tasks-config/common/webserver.js

```diff
@@ -3,28 +3,35 @@
 
     const gulpConnect = require('gulp-connect');
     const gulpOpen = require('gulp-open');
+    const args = require('yargs').argv;
 
     module.exports = function (gulp, configuration) {
 
         gulp.task('webserver', webServerTask);
         gulp.task('webserver:open-browser', openBrowserTask);
 
-        function webServerTask() {
+        function webServerTask () {
+            if (!args.secure) {
+                console.info('You can start an HTTPS server by running gulp with the --secure flag');
+            }
+
             return gulpConnect.server({
                 root: [configuration.projectReleasePath],
                 port: configuration.projectConfig.build.server_port,
-                livereload: true
+                livereload: true,
+                https: args.secure !== null && args.secure !== undefined
             });
         }
 
-        function openBrowserTask() {
+        function openBrowserTask () {
+            const protocol = (args.secure !== null && args.secure !== undefined) ? 'https' : 'http';
```

Comment:
well, I was not considering gulp --secure=hello as valid command; but technically, I agree.